redbrick/export/public.py

```
@@ -353,6 +353,16 @@ async def _download_task_items(
 
             to_presign = []
             local_files = []
+
+            if any(self.contains_altadb_item(item_list) for item_list in items_lists):
+                logger.warning(
+                    f"Task {task.get('taskId')} contains items from AltADB. Download ignored for the task."
+                )
+                # Delete if the task directory is empty
+                if os.path.exists(task_dir) and not os.listdir(task_dir):
+                    os.rmdir(task_dir)
+                return task, series_dirs
```

what is the significance of series_dirs ?

---

Should it be returning an empty list?

---

you tell me 😺

---

I think we should return the empty list in that case. The empty list will avoid any computation when `dcm_to_nii` will be True because it need not be processed.

---

can we raise an Exception instead?

---

Should we?
That'll show an explicit `Exception`.

---

catch it in the calling method

---

It is already handled.

---

that's great.
so let's just throw proper error instead of worrying about what to return